build-logic/src/main/kotlin/com.hedera.hashgraph.jpms-module-dependencies.gradle.kts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2016-2022 Hedera Hashgraph, LLC
+ *
+ * This software is the confidential and proprietary information of
+ * Hedera Hashgraph, LLC. ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement yaou entered into
+ * with Hedera Hashgraph.
+ *
+ * HEDERA HASHGRAPH MAKES NO REPRESENTATIONS OR WARRANTIES ABOUT THE SUITABILITY OF
+ * THE SOFTWARE, EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED
+ * TO THE IMPLIED WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE, OR NON-INFRINGEMENT. HEDERA HASHGRAPH SHALL NOT BE LIABLE FOR
+ * ANY DAMAGES SUFFERED BY LICENSEE AS A RESULT OF USING, MODIFYING OR
+ * DISTRIBUTING THIS SOFTWARE OR ITS DERIVATIVES.
+ */
+
+plugins {
+    id("org.gradlex.java-module-dependencies")
+}
+
+javaModuleDependencies {
+    warnForMissingVersions.set(false) // do not expect versions in catalog
```

Comment:
What does that flag do?

Reply:
As soon as a version catalog is available, the module-dependencies plugin automatically looks for versions in it. If it can't find a version, it warns. This turns this behavior off.

We need it, because we still use the catalog right now for a few things unrelated to modules. If we won't use the catalog (in settings.gradle.kts) at all, the warnings would be turned off by default.

I should add more descriptive comments to all of these things in a follow up PR.